cmd/flux/export.go

```diff
@@ -115,7 +115,7 @@ func printExport(export interface{}) error {
 		return err
 	}
 	rootCmd.Println("---")
-	rootCmd.Println(resourceToString(data))
+	rootCmd.Print(resourceToString(data))
```

Comment:
Seeing this together with `resourceToString`, it might actually be better to add a `data = bytes.TrimSpace(data)` to the function below and keep this in place.

This would simply ensure that even if the generating function makes a mistake (by either forgetting to add a newline, or adding to many). We always return just a single newline at the end.

Reply:
Great suggestion 👍